lib/k8s/client.ex

```diff
@@ -180,7 +180,7 @@ defmodule K8s.Client do
         path_params: [namespace: "test", name: "nginx"]
       }
 
-    Get the nginx deployment in the default namespace:
+    Get the nginx deployment in the default namespace using atom name.  e.g. `name: :deployment`:
```

Comment:
it's the _kind_ actually, that is passed as atom in this example (e.g. `:deployment`), not the way you wrote in the documentation.

Reply:
Indeed!  I updated and pushed